PowerEditor/installer/nativeLang/french.xml

```diff
@@ -25,7 +25,7 @@ Translation note:
 				</Entries>
 				<!-- Sub Menu Entries -->
 				<SubEntries>
-					<Item subMenuId="file-openFolder" name="Ouvrir le chemin du fichier avec :"/>
+					<Item subMenuId="file-openFolder" name="Utiliser le chemin pour ouvrir :"/>
```

Comment:
It should be `<Item subMenuId="file-openFolder" name="Ouvrir le chemin du fichier avec :"/>`